src/components/ContentListTable/components/EditContentModal/EditContentModal.test.tsx

```diff
@@ -84,7 +90,7 @@ it('Open, confirming values, edit an item, enabling Save button', async () => {
   }
   expect(queryByText('test gpg key')).toBeInTheDocument();
```

Comment:
GPG key? (see my comment for line 228)

Reply:
this is test code only. Won't be seen by the end user, but I'll get in the habit of doing GPG in the future :)